packages/db/src/data-source.ts

```
@@ -1,47 +1,25 @@
 dotenv.config({ path: path.join(packageRoot, ".db.env") });
 
-const databaseHost = process.env.DATABASE_HOST ?? "localhost";
-const databasePort = process.env.DATABASE_PORT ?? "5432";
-const databaseUsername = process.env.DATABASE_USERNAME ?? process.env.POSTGRES_USER;
-const databasePassword = process.env.DATABASE_PASSWORD ?? process.env.POSTGRES_PASSWORD;
+const databaseHost = process.env.DATABASE_HOST;
+const databasePort = process.env.DATABASE_PORT;
+const databaseUsername = process.env.DATABASE_USERNAME;
+const databasePassword = process.env.DATABASE_PASSWORD;
+
+if (!process.env.DATABASE_URL && databaseHost && databasePort && databaseUsername && databasePassword) {
```

ESLint

[error] Replace `!process.env.DATABASE_URL·&&·databaseHost·&&·databasePort·&&·databaseUsername·&&·databasePassword` with `⏎··!process.env.DATABASE_URL·&&⏎··databaseHost·&&⏎··databasePort·&&⏎··databaseUsername·&&⏎··databasePassword⏎`

(prettier/prettier)